swagger.json

```diff
@@ -654,6 +654,60 @@
             "headers": {}
           }
         }
+      },
+      "patch": {
+        "description": "A line manager can respond to assigned trip requests",
+        "summary": "accept or reject a trip request",
+        "tags": [
+          "Trips"
+        ],
+        "operationId": "RespondTrip",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [{
+            "name": "token",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "token identify that a manager has signed in"
+          },
+          {
+            "in": "path",
+            "name": "tripId",
+            "type": "string",
+            "required": true,
+            "description": "tripId of a trip to be accepted or rejected"
+          },
+          {
+            "name": "Body",
+            "in": "body",
+            "required": true,
+            "description": "line manager can accept or reject a trip request",
```

Comment:
Is this the description of the body?

Reply:
Yes ofcourse,  It's the description of the body to be passed to the endpoint

Reply:
Describe what should be passed in the body instead of repeating yourself. Because this has been already mentioned above.

Reply:
I already did that